packages/orchestration/test/examples/stake-ica.contract.test.ts

```diff
@@ -132,6 +137,56 @@ test('makeAccount, getAddress, getBalances, getBalance', async t => {
   t.snapshot(bootstrap.storage.data.entries(), 'accounts in vstorage');
 });
 
+test('delegate, undelegate, redelegate, withdrawReward', async t => {
+  const { bootstrap } = await commonSetup(t);
+  const { publicFacet } = await startContract(bootstrap);
+  const account = await E(publicFacet).makeAccount();
+
+  // XXX consider building a mock bank into remote chains. for now, assume
+  // newly created accounts magically have tokens.
+  const validatorAddr = {
+    address: 'cosmosvaloper1test' as const,
+    chainId: 'cosmoshub-4',
+    addressEncoding: 'bech32' as const,
+  };
+  const delegation = await E(account).delegate(validatorAddr, {
+    denom: 'uatom',
+    value: 10n,
+  });
+  t.is(delegation, undefined, 'delegation returns void');
+
+  // TODO, fixme!
```

Comment:
I was reviewing by commit so I drafted a comment asking when, then I saw the fix in a subsequent PR.

I support this style of queuing up work for other commits in the PR. Consider being more specific for the reader, e.g.  `FIXME in this PR`

Reply:
Acknowledged for the future